core/src/test/scala/unit/kafka/integration/KafkaServerTestHarness.scala

```diff
@@ -54,11 +58,18 @@ trait KafkaServerTestHarness extends ZooKeeperTestHarness {
 
   protected def securityProtocol: SecurityProtocol = SecurityProtocol.PLAINTEXT
   protected def trustStoreFile: Option[File] = None
+  def clusterAclArgs: Array[String] = Array("--authorizer-properties", 
+                                            s"zookeeper.connect=$zkConnect",
+                                            s"--add",
+                                            s"--cluster",
+                                            s"--allow-principal=$kafkaPrincipal")
```

Comment:
Would it be better to move this to EndToEndAuthorizationTest since it's specific to authorization?